.circleci/config.yml

```diff
@@ -1,24 +1,37 @@
 version: 2
 
 jobs:
-
   build:
     docker:
-      - image: python:3.7-alpine
+      - image: circleci/python:3.7
     steps:
       - checkout
-      - run: apk add git
-      - run: pip install -r requirements.txt
-      - run: echo 'Run the tests....'
+
+      - run: sudo chown -R circleci:circleci /usr/local/bin
+      - run: sudo chown -R circleci:circleci /usr/local/lib/python3.7/site-packages
+      - restore_cache:
+          key: deps9-{{ .Branch }}-{{ checksum "Pipfile.lock" }}
```

Comment:
set up caching (from a the circle docs) on the python dependencies. That cache should get busted every time the lock files changes, which means that the dependencies changed